src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -95,7 +101,8 @@ export class SkyColorpickerInputDirective
     private elementRef: ElementRef,
     private renderer: Renderer,
     private service: SkyColorpickerService,
-    private resourcesService: SkyLibResourcesService
+    private resourcesService: SkyLibResourcesService,
+    private inj: Injector
```

Comment:
We should avoid abbreviated variable names, so there's less room for confusion later on.
`private injector: Injector`

Reply:
Will do